hw/ip/kmac/doc/checklist.md

```diff
@@ -129,8 +129,9 @@ Coverage      | [SIM_COVERAGE_MODEL_ADDED][]          | Done        |
 Code Quality  | [TB_LINT_SETUP][]                     | Done        |
 Integration   | [PRE_VERIFIED_SUB_MODULES_V1][]       | Done        |
 Review        | [DESIGN_SPEC_REVIEWED][]              | Done        |
-Review        | [DV_PLAN_TESTPLAN_REVIEWED][]         | Not Started |
```

Comment:
TB_GEN_AUTOMATED should be N/A 
FPV_MAIN_ASSERTIONS_PROVEN seems only applicable for FPV, it should be also N/A. @cindychip LMK if it's not right

Reply:
yup, agree with weicai

Reply:
Hmm ok. I set TB_GEN_AUTOMATED to Done because of multiple parameter configs being verified, but I can change both to N/A as well. thanks for the comment!